src/specs/models/processing-config-bodies/ProcessingConfigDimensionalityReduction.v1.yaml

```diff
@@ -0,0 +1,32 @@
+title: Dimensionality Reduction
```

Comment:
this should be called Data Integration, so that it is consistent with what we have in the UI

Reply:
Again, this is what it's called right now and how the default experiments are structured. If we need to change it, it should be a separate ticket probably.

Reply:
Yup, it should be part of persisting data integration config to DynamoDB